api/kyverno/v1/utils.go

```diff
@@ -9,9 +13,21 @@ import (
 
 func FromJSON(in *apiextv1.JSON) apiextensions.JSON {
 	var out apiextensions.JSON
-	if err := apiextv1.Convert_v1_JSON_To_apiextensions_JSON(in, &out, nil); err != nil {
-		log.Log.Error(err, "failed to convert JSON to interface")
+
+	if in != nil {
+		var i interface{}
+		if len(in.Raw) > 0 && !bytes.Equal(in.Raw, []byte(`null`)) {
+			d := json.NewDecoder(strings.NewReader(string(in.Raw)))
+			d.UseNumber()
+			if err := d.Decode(&out); err != nil {
+				log.Log.Error(err, "failed to convert JSON to interface")
+			}
```

Comment:
@samj1912 @realshuting - I'm not sure if this would be a breaking change, but I couldn't find another fix for the unmarshalling error caused by the public key modulus being out of `float64` range. Might need help resolving this if possible with a better approach.

Reply:
And I guess this broke 10 tests 🥲

Reply:
Here's the error due to which I had to introduce these changes:
<img width="643" alt="image" src="https://user-images.githubusercontent.com/37282098/191856587-c049350f-9854-40aa-88ff-8b86e662052f.png">

@samj1912 - Through my tests I've concluded that using `github.com/smallstep/zcrypto` would be a better option since its X.509 implementation stores the public key modulus as hex instead of decimal, and it also works with `json.Unmarshal`, so I guess we need to make a choice between using this external library and the standard one 😓